nips2018/deep_learning/README.md

```diff
@@ -4,11 +4,16 @@ This example illustrates training of mnist on an hypersphere manifold.
 
 The installation of this example is currently quite involved. You will need to:
 
-- Update tensorflow from a patch file in this repository (re-compiling would be a nightmare - this solution is a hack but it is the least painful path for now.). You will also need a fairly recent version of tensorflow (1.8 at least).
+- Update tensorflow from a patch file in this repository (re-compiling would be a nightmare - this solution is a hack but it is the least painful path for now.). You will also need a fairly recent version of tensorflow (1.8 at least). BEWARE! This *will* alter your version of tensorflow - and might break it. You need to
 - Update keras (the following steps are just if you've never installed keras - you might want to uninstall it before re-installing this version):
```

Comment:
Rephrase what is inside the parenthesis? it is not so clear.
Maybe something like:
If you've never installed Keras, follow the steps below. If you've installed Keras before, we recommend that you uninstall it before installing this version.
?